types/node-geocoder/index.d.ts

```diff
@@ -0,0 +1,92 @@
+// Type definitions for node-geocoder 3.19
+// Project: https://github.com/nchaulet/node-geocoder#readme
+// Definitions by: Krzysztof Rosinski <https://github.com/rosek86>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare namespace node_geocoder {
+    type Providers =
+        'google' | 'here' | 'freegeoip' |
+        'datasciencetoolkit' | 'openstreetmap' |
+        'locationiq' | 'mapquest' | 'openmapquest' |
+        'agol' | 'tomtom' | 'nominatimmapquest' |
```

Comment:
Shouldnt this also have mapzen:  https://github.com/nchaulet/node-geocoder/blob/master/lib/geocoderfactory.js#L84